docs/frost-bookends.md

```diff
@@ -0,0 +1,67 @@
+# frost-bookends <br />
+
+ * [API](#api)
+ * [Examples](#examples)
+
+## API
+
+| Attribute   | Type | Value | Description |
+| ----------- | ---- | ----- | ----------- |
+| `content` | `component` | | **required** - component to render as the content (between header and footer) |
```

Comment:
Sounds good. I was thinking we couldn't do block b/c of the need for header/footer, but I guess it makes sense to have the content be the inner block stuff.